test/pedestal/swagger/core_test.clj

```diff
@@ -69,7 +69,7 @@
                          (coerce-params) (validate-response)
                          auth-middleware]
      {:get get-handler}
-     ["/:id" ^:interceptors [id-middleware]
+     ["/x/:id" ^:interceptors [id-middleware]
```

Comment:
conflict with the new router - using the linear search router this is not necessary